filebeat/input/journald/input_test.go

```diff
@@ -171,6 +171,11 @@ func TestCompareGoSystemdWithJournalctl(t *testing.T) {
 		t.Fatalf("expecting %d events, got %d", len(goldenEvents), len(events))
 	}
 
+	// After the golden events were generated we changed some field names
+	// to better align with ECS, so we do the same conversion in the golden
+	// events.
+	updateGoldenEvents(goldenEvents)
```

Comment:
Is there a reason to do this in code instead of updating the golden file? I don't see any other tests that reference it

Reply:
Yes, I wanted to keep the golden files untouched as they're the snapshot from the previous implementation, the one using go-systemd.

However I'm not strongly set on it, I can remove this function and edit the file if you believe it's better.

Reply:
Hmm, I think it is valuable to have done an explicit test against the old version, but now that you have I think the golden files should be updated before merge, a code workaround would definitely make things more confusing for the next person who has to regenerate them :-)